cfme/middleware/provider.py

```diff
@@ -125,3 +127,19 @@ def num_datasource_ui(self, reload_data=True):
         if reload_data:
             self.summary.reload()
         return self.summary.relationships.middleware_datasources.value
+
+    @variable(alias='ui')
+    def is_refreshed(self, reload_data=True):
+        if reload_data:
+            self.summary.reload()
+        if re.match('Success.*Minute.*Ago', self.summary.status.last_refresh.value):
```

Comment:
@hhovsepy As we know this is `text` we can use `self.summary.status.last_refresh.text_value`